lib/solarwinds_apm/patch/tag_sql/annotate_traceparent.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module SolarWindsAPM
+  module Patch
+    module TagSql
+      module AnnotateTraceparent
+        def self.generate_traceparent
+          span_context = ::OpenTelemetry::Trace.current_span.context
+          return '' if span_context == ::OpenTelemetry::Trace::SpanContext::INVALID
+
+          trace_flag = span_context.trace_flags.sampled? ? '01' : '00'
+          return '' if trace_flag == '00'
```

Comment:
Seems it should be enough to just `return '' unless span_context.trace_flags.sampled?` ?

Reply:
afaik, I don't there is function for "get traceparent as string" , but I simplified to `return '' unless pan_context.trace_flags.sampled?`, and hardcoded trace_flag to `01` for sampled traceparent

Reply:
How about this one https://github.com/solarwinds/apm-ruby/blob/main/lib/solarwinds_apm/support/utils.rb#L24 :)

Reply:
also wondering whether the `AnnoteTraceparent` module is actually needed if the util method can be used.

Reply:
Sure, updated and removed the AnnoteTraceparent module